csp.ipynb

```diff
@@ -950,15 +944,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The _ ___init___ _ method takes only one parameter **n** the size of the problem. To create an instance we just pass the required n into the constructor."
+    "The _ ___init___ _ method takes only one parameter **n** i.e. the size of the problem. To create an instance, we simply just pass the required value of n into the constructor."
```

Comment:
The 'simply' here is redundant.
'simply' and 'just' mean the same thing in this context.

Reply:
Agreed. Thanks.